<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MatplotlibWidget" name="mplwidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>521</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantum Wire Conductance Measurement</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>143</width>
      <height>383</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Gate Voltage</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="gate_display"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="conductanceLabel">
               <property name="text">
                <string>Conductance</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="conductance"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="timeLabel">
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="time"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="tempLabel">
               <property name="text">
                <string>Temperature</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="temp"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="pointsLabel">
               <property name="text">
                <string>Step Size</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="stepSize">
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="maxGateLabel">
               <property name="text">
                <string>Max gate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="maxGate">
               <property name="minimum">
                <double>-99.000000000000000</double>
               </property>
               <property name="value">
                <double>-2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="delayLabel_2">
               <property name="text">
                <string>Delay</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="delay">
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Focus</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Lower Limit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="windowlower">
               <property name="minimum">
                <double>-5.000000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Upper Limit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="windowupper">
               <property name="minimum">
                <double>-5.000000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Step Size</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="windowstep">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="goButton">
         <property name="text">
          <string>Go!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>mplwidget</zorder>
   <zorder>label_6</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
